docs/writing-tests/index.md

```diff
@@ -40,19 +40,19 @@ expectations:
   for tips on how to write great rendering tests. There are a few different
   ways to write rendering tests:
 
-  * [Reftests](reftests) should be used to test rendering and layout. They
+  * [Reftests](reftests) are great for testing rendering and layout. They
     consist of two or more pages with assertions as to whether they render
     identically or not.
 
-  * [Visual tests](visual) should be used for checking rendering where there is
-    a large number of conforming renderings such that reftests are impractical.
+  * [Visual tests](visual) are great for checking rendering where there is a
```

Comment:
(They're never great. They're practically manual tests. Maybe we should just list them down by manual?)